proposals/0258-property-wrappers.md

```diff
@@ -285,7 +375,7 @@ class Foo {
 
 Many Cocoa classes implement value-like objects that require explicit copying.
 Swift currently provides an `@NSCopying` attribute for properties to give
-them behavior like Objective-C's `@property(copy)`, invoking the `copy` method
+them behave like Objective-C's `@property(copy)`, invoking the `copy` method
```

Comment:
`behave` looks less grammatically correct than `behavior` here.

Reply:
Whoops, fixed it back.